.travis.yml

```diff
@@ -9,42 +9,80 @@
 ##
 
 sudo: required
+
 dist: trusty
+
 branches:
   only:
     - master
-matrix:
-  fast_finish: true
 
-env:
-  - TARGET="Style"
-  - TARGET="Registers"
+matrix:
+  include:
+    - os: osx
```

Comment:
Also, move the Darwin builds at the end of the block, after the LLDB unit test ones please.